lib/app/components/atoms/tab_button.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class TabButton extends StatelessWidget {
+  const TabButton(
+      {super.key,
+      required this.chipTitle,
+      required this.isEnable,
+      required this.onPressed});
```

Comment:
```suggestion
      required this.onPressed,
      });
```
細かくてすいませんが、コンマつけて改行お願いします。
ところどころ改行されてないところがあるので、
コマンド叩いて全体的にフォーマットしてみてもいいかもしれないですね。
[Code formatting](https://docs.flutter.dev/development/tools/formatting)
```
flutter format
```

Reply:
@king-kazu39 
VSCodeで自動フォーマットの設定をしているので flutter format は必要ないかと思っていました。両方使われてますか？
コンマを打たないと改行をしてくれないということですね。つけるようにします。

Reply:
flutter formatしても特にコンマが付くということはないようです。
コンマをつけた後の保存時の自動フォーマットで自動的に改行されるようでした。
対応いたしました。